patternfly/patternfly-elements
# fix(docs): import map module versions and query assigned elements

## What I did

1. `docs/_data/importMaps.cjs` now reads module versions from its package.json file.  
2. included decorator `@queryAssignedElements` with query-assigned-elements.js

## How to test

Modify a component to include @queryAssignedElements decorator, run `npm run watch:docs` functionality should work in generated docs demo.
